static/js/billing/upgrade.js

```diff
@@ -71,6 +72,18 @@ export const initialize = () => {
 
     helpers.show_license_section($("input[type=radio][name=license_management]:checked").val());
     helpers.update_charged_amount(prices, $("input[type=radio][name=schedule]:checked").val());
+
+    if (page_params.demo_organization_scheduled_deletion_date) {
+        tippy("[data-tippy-content]", {
+            // Same defaults as set in our tippyjs module.
+            maxWidth: 300,
+            delay: [100, 20],
+            animation: false,
+            touch: ["hold", 750],
+            placement: "bottom",
+        });
```

Comment:
Couldn't we just make this library initializing unconditional?  

I guess there's a question of whether we want to be doing this in something like `portico.js` rather than a billing-specific page; @amanagr thoughts?

Reply:
True, let's just initialize this in portico.

Reply:
OK, so let's split this out as an independent prep commit that adds Tippy support for portico pages; perhaps @amanagr and @laurynmm can coordinate on having that happen.